README.md

```diff
@@ -1,4 +1,40 @@
-finder-pipe
-===========
+# finder-pipe
 
-Utiltity tools for Archive/Directory/File.
+The utiltity tools for Archive/Directory/File.
+
+## Installation
+
+- [ ] Describe the installation process
+
+## Usage
+
+This example is some workflow.
```

Comment:
very similar to gulp.

I want to gulp, am not?

This process to be harmonious with `Promise`.
I think Node stream is unmatch with the process. Specially in archieving directory.